community/how-to-release.md

```diff
@@ -287,7 +289,8 @@ $ svn commit -m "prepare for ${release_version} ${rc_version}"
 - You will get a link like `https://repository.apache.org/content/repositories/orgapacheinlong-{staging-id}` on the Summary page if all checks passed. This link will be posted in the vote email, so, keep it safe.
 
 :::caution
-Close operation may fail, you should check the causes and fix them.
+- Close operation may fail, you should check the causes and fix them.
+- The key you upload before may not be synchronized among keyserver, please upload it to keyserver.ubuntu.com or keys.openpgp.org if staging fail by error ![img.png](images/staging-error.png) 
```

Comment:
——>
- The close operation may not work; it's better to check and fix the causes.
- The key you uploaded before may not be synchronized among the keyserver. Please upload it to `keyserver.ubuntu.com` or `keys.openpgp.org` if there is a `Signature Validation` event.